test/expression.js

```diff
@@ -1235,6 +1235,20 @@ test('it will work with #139 issue case', function(t) {
   t.end();
 });
 
+test('it will work with #271 issue case', function(t) {
```

Comment:
Given that this test fails for Node v 12 only in the CI but that this PR is a fix for the issue for newer versions of Node, one options would be the following.  It's not elegant but it is at least an improvement that fixes this issue in newer versions of Node and breaks nothing, according to the existing unit tests, in Node 12.

```suggestion
test('it will work with #271 issue case for Node v > 12', function(t) {
  if(process.versions.node.startsWith('12.')) {
      t.end();
      return;
  }
```